moped-editor/src/utils/activityLogFormatters/mopedProjectActivity.js

```diff
@@ -112,6 +138,22 @@ export const formatProjectActivity = (change, lookupList) => {
       };
     }
 
+    if (updatedFullName) {
+      return {
+        changeIcon,
+        changeText: [
+          {
+            text: "Changed project name to ",
+            style: null,
+          },
+          {
+            text: changeData.new["project_name_full"],
+            style: "boldText",
+          },
+        ],
+      };
+    }
+
```

Comment:
@amenity Here's the update, if someone changes both the primary name and the secondary name at the same time, it shows the full name. 

<img width="1302" alt="Screenshot 2024-04-11 at 9 39 04 PM" src="https://github.com/cityofaustin/atd-moped/assets/12474808/fbc0a2d4-8606-495b-9194-6a89fa0035e9">

Another option would be to have it say "Updated project name and secondary name" without including the changed text.